### PR TITLE
Fix: Clarify the wording of the release issue template

### PR DESCRIPTION
**Pull request by graywolf336**

### CODE REVIEW

File: .github/issue-templates/release.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-For any regression, open a new issue and link to this one.
+When you find a bug that is a regression, please open a new issue and link it to this one.
 
 
 ## Before Release - Preparation - 1 business day before the day 20th
 - [x] Create the issue to track the release progress
-- [ ] Define the highlights from release PRs as suggestion to be included on Blog Post
-- [ ] Talk to the Marketing Team about the Blog Post release
-- [ ] Talk to the Documentation Team about the Docs release
+- [ ] Define the highlights from release PRs as suggestion to be included on Blog Post (as a new comment to this issue with links to the PRs)
```
Comment (geekgonecrazy) [inline .github/issue-templates/release.md]: Hasn't this actually been done on the website repo the past couple of releases?

Comment (graywolf336) [reply .github/issue-templates/release.md]: I wasn't aware of that. @rodrigok can you comment here?

Comment (geekgonecrazy) [reply .github/issue-templates/release.md]: https://github.com/RocketChat/rocketchat.github.io/issues/162 this is what has been happening

Comment (rodrigok) [reply .github/issue-templates/release.md]: @graywolf336 @geekgonecrazy Yes, we are opening a new issue on the website repo to list and discuss the highlights, we should edit a put the link here to be more clear

**Pull request merged**